recipes/allegro5/all/conanfile.py

```diff
@@ -0,0 +1,165 @@
+        self._patch_addon("addons", "${PHYSFS_INCLUDE_DIR}", "${physfs_INCLUDE_DIR}")
+        self._patch_addon("addons", "${PHYSFS_LIBRARY}", "${physfs_LIBRARY}")
+        self._patch_addon("physfs", "${PHYSFS_INCLUDE_DIR}", "${physfs_INCLUDE_DIR}")
+        self._patch_addon("physfs", "${PHYSFS_INCLUDE_FILES}", "${physfs_INCLUDE_FILES}")
+        self._patch_addon("addons", "find_package_handle_standard_args(PHYSFS DEFAULT_MSG\n        PHYSFS_LIBRARY PHYSFS_INCLUDE_DIR)", "")
+
+        self._patch_addon(None, "find_package(X11)", "find_package(X11 REQUIRED)\nset(X11_LIBRARIES ${X11_LIBRARIES} xcb dl)")
+        self._patch_addon("addons", "run_c_compile_test(\"${FREETYPE_TEST_SOURCE}\" TTF_COMPILES)", "")
+        self._patch_addon("addons", "run_c_compile_test(\"${FREETYPE_TEST_SOURCE}\" TTF_COMPILES_WITH_EXTRA_DEPS)", 
+            '''find_package(Brotli REQUIRED)
+               list(APPEND FREETYPE_STATIC_LIBRARIES "${Brotli_LIBRARIES}")
+               run_c_compile_test("${FREETYPE_TEST_SOURCE}" TTF_COMPILES_WITH_EXTRA_DEPS)''')
+        self._patch_addon(None, "set(INSTALL_PKG_CONFIG_FILES true)", "set(INSTALL_PKG_CONFIG_FILES false)")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        self._cmake.definitions["SHARED"] = False
+        self._cmake.definitions["WANT_STATIC_RUNTIME"] = "MT" in msvc_runtime_flag(self) if self._is_msvc else False
+        self._cmake.definitions["CMAKE_BUILD_TYPE"] = self.settings.build_type
+        self._cmake.definitions["PREFER_STATIC_DEPS"] = True
+        self._cmake.definitions["WANT_DOCS"] = False
+        self._cmake.definitions["WANT_DOCS_HTML"] = False
+        self._cmake.definitions["WANT_EXAMPLES"] = False
+        self._cmake.definitions["WANT_FONT"] = True
+        self._cmake.definitions["WANT_MONOLITH"] = True
+        self._cmake.definitions["WANT_TESTS"] = False
+        self._cmake.definitions["WANT_DEMO"] = False
+        self._cmake.definitions["WANT_RELEASE_LOGGING"] = False
+        self._cmake.definitions["WANT_VORBIS"] = True
+        self._cmake.definitions["WANT_MP3"] = True
+        self._cmake.definitions["WANT_OGG_VIDEO"] = True
+
+        self._cmake.definitions["FREETYPE_ZLIB"] = self.options["freetype"].with_zlib
+        self._cmake.definitions["FREETYPE_BZIP2"] = self.options["freetype"].with_bzip2
+        self._cmake.definitions["FREETYPE_PNG"] = self.options["freetype"].with_png
+        self._cmake.definitions["FREETYPE_HARFBUZZ"] = False
+            
+        self._cmake.definitions["CMAKE_CXX_FLAGS"] = "/wd4267 /wd4018" if self._is_msvc else "-Wno-unused-variable -w"
+        
+        self._cmake.configure(source_folder=self._source_subfolder, build_folder=self._build_subfolder)
```

Comment:
```suggestion
        self._cmake.configure(build_folder=self._build_subfolder)
```

Need to use cmake wrapper. As example: https://github.com/conan-io/conan-center-index/blob/master/recipes/aaplus/all/CMakeLists.txt